keras_hub/src/models/roformer_v2/roformer_v2_text_classifier.py

```diff
@@ -0,0 +1,122 @@
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.bert.bert_text_classifier import BertTextClassifier
+from keras_hub.src.models.roformer_v2.roformer_v2_backbone import (
+    RoformerV2Backbone,
+)
+from keras_hub.src.models.roformer_v2.roformer_v2_text_classifier_preprocessor import (  # noqa: E501
+    RoformerV2TextClassifierPreprocessor,
+)
+
+
+@keras_hub_export(
+    [
```

Comment:
You only need this list form when you are defining an alias. For all `keras_hub_export` with only one symbol name, delete the `[]`